core/controllers/topic_editor_test.py

```diff
@@ -92,6 +93,55 @@ def test_access_topic_editor_page(self):
             self.assertEqual(response.status_int, 200)
             self.logout()
 
+            # Assign TOPIC_MANAGER_USERNAME as a topic_manager for topic_id.
+            topic_services.assign_role(
+                self.admin, self.topic_manager, topic_domain.ROLE_MANAGER,
+                self.topic_id)
+            # Check that a topic manager for a topic can access and edit in the
+            # editor page.
+            self.login(self.TOPIC_MANAGER_EMAIL)
+            response = self.testapp.get(
+                '%s/%s' % (feconf.TOPIC_EDITOR_URL_PREFIX, self.topic_id))
+            self.assertEqual(response.status_int, 200)
+            self.logout()
+
+            # Create a new topic, and assign another user as its topic manager.
+            topic_id_2 = topic_services.get_new_topic_id()
+            self.save_new_topic(
+                topic_id_2, self.admin_id, 'Name', 'Description',
+                [], [], [])
+            self.signup('topicmanager2@example.com', 'topicmanager2')
+            topic_manager_id_2 = self.get_user_id_from_email(
+                'topicmanager2@example.com')
+            self.set_topic_managers(['topicmanager2'])
+            topic_manager_2 = user_services.UserActionsInfo(
+                topic_manager_id_2)
+            topic_services.assign_role(
+                self.admin, topic_manager_2, topic_domain.ROLE_MANAGER,
+                topic_id_2)
+
+            # Verify that the created user can edit the assigned topic.
```

Comment:
"the created user" is quite vague; every user in this test is created. Perhaps "the second user" or "the second topic manager", and change "the assigned topic" to "their assigned topic".

In general, it's worth being specific/precise in comments, since they'll live on in the codebase for quite a long time.

Reply:
Done.